packages/iov-bns-governance/src/index.ts

```diff
@@ -0,0 +1,137 @@
+import { As } from "type-tagger";
+
+import { Address, Amount, ChainId, PostTxResponse } from "@iov/bcp";
+import { BnsConnection, Electorate, Fraction, Proposal, VoteOption } from "@iov/bns";
+import { UserProfile } from "@iov/keycontrol";
+
+type CommitteeId = number & As<"committee">;
+
+/* Constants to be finalised once the committees are actually on-chain */
+export const COMMITTEE_IDS = {
+  ["testnet" as ChainId]: {
```

Comment:
as far as I know, object key types can only be `string` or `number`, so we will have trouble creating a type for this literal on other places. But plain strings instead of `ChainId` are fine.